pkg/ebpf/net_events.go

```diff
@@ -21,8 +21,9 @@ type EventMeta struct {
 }
 
 type CaptureData struct {
-	PacketLen      uint32 `json:"pkt_len"`
-	InterfaceIndex uint32 `json:"if_index"`
+	PacketLength      uint32 `json:"pkt_len"`
+	InterfaceIndex    uint32 `json:"if_index"`
+	PacketStartOffset uint32 `json:"pkt_start_off"`
```

Comment:
Your'e mixing the data received from bpf code with variables used to parse it.
Please don't do that.